src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  padding-top: 12px;
+}
+
+.field {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 900;
+  height: 100%;
+  width: 100%;
+  border: none;
+  outline: none;
+}
+
+.fieldBig {
```

Comment:
use snake-case, not the camelCase for class names

Reply:
done